public/js/app/templates/bookmarkletTemplate.handlebars

```diff
@@ -99,24 +102,40 @@
 {{else}}
 
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-xs-12">
       <h1>Share on FreeFeed</h1>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-xs-12">
+      <label for="bookmarklet-recipients">To:</label><br/>
+      <select id="bookmarklet-recipients" multiple="multiple" class="p-recipients-select">
+        <option selected="selected" value="{{controller.session.currentUser.username}}" class="p-recipients-selected">My feed</option>
+        {{#each feed in controller.session.currentUser.feeds}}
+          <option value="{{feed.user.username}}" class="p-recipients-option">{{feed.user.screenName}} ({{feed.user.username}})</option>
+        {{/each}}
+      </select>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-xs-12">
+      <label for="bookmarklet-title">Title:</label><br/>
       <textarea id="bookmarklet-title" name="title">{{controller.title}}</textarea>
     </div>
   </div>
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-xs-12">
       <div id="bookmarklet-image" class="no-image">No image<br/> selected</div>
     </div>
   </div>
   <div class="row">
-    <div class="col-md-12">
+    <div class="col-xs-12">
       <label for="bookmarklet-comment">Comment (optional)</label><br/>
       <textarea id="bookmarklet-comment" name="comment">{{controller.comment}}</textarea>
     </div>
   </div>
   <div class="row">
-    <div class="col-md-12 text-right">
+    <div class="col-xs-12 text-right">
```

Comment:
Fixed, just thought full width "post" button looks more consistent in bookmarklet.